### ISSUE TITLE
Accept counts

### ISSUE THREAD

**Issue by theHamsta** [nvim-treesitter/nvim-treesitter-textobjects#8]
Would be nice if the move commands could accept counts. And also the select commands could accept counts to select an outer textobject in a nested structure.

**Comment by sanga** [comment]
as in `v2af` selecting 2 functions? If so, yes, that would be very nice (I came here to file that feature request)

**Comment by theHamsta** [comment]
> as in v2af selecting 2 functions? If so, yes, that would be very nice (I came here to file that feature request)

Yes, either that but maybe also to select the outer ones of two nested functions.

**Comment by singlexyz** [comment]
> Would be nice if the move commands could accept counts. And also the select commands could accept counts to select an outer textobject in a nested structure.

Select outer and inner, I think here is.
https://github.com/nvim-treesitter/nvim-treesitter#incremental-selection

**Comment by theHamsta** [comment]
Will add the count option soon, this will also make the plugin a lot faster...

**Comment by jameswalmsley** [comment]
Any news on implementation of this?

**Comment by danielnehrig** [comment]
would this also cover the use case for jumping to a second match for instance deleting the second argument in a function
d2]a
